python/pyspark/worker.py

```diff
@@ -91,10 +92,16 @@ def verify_result_length(*a):
 
 
 def wrap_grouped_map_pandas_udf(f, return_type):
-    def wrapped(*series):
+    def wrapped(key_series, value_series):
         import pandas as pd
+        argspec = inspect.getargspec(f)
```

Comment:
should this also do `getfullargspec` for py3 like in udf.py?
maybe it would be useful to put a function in util.py, what do you guys think?

Reply:
Good point. Let me take a look at that.